config/routes.rb

```diff
@@ -16,6 +16,7 @@
 
       resources :stories, except: [:new, :edit] do
         get 'random', on: :collection
+        get 'search', on: :collection
```

Comment:
Hrmm... yeah, remind me what this is for, if we're just overriding the `?text` filter.

Reply:
+1 on separate endpoint, at least to start.